.github/workflows/codegen-build-test.yml

```diff
@@ -61,5 +64,16 @@ jobs:
         run: ./scripts/ci_steps/codegen_sdk.sh
       - name: Build SDK with Unit Tests
         run: swift build --build-tests
+      - name: Create AWS config directory
```

Comment:
Let's add a comment here about why we're going to the trouble of setting up these credentials in files, so future us doesn't forget about the manipulation of env vars in the credential resolver tests.